File: component_core/src/main/java/com/ashlikun/baseproject/libcore/javabean/KeyAndValus.java
package com.ashlikun.baseproject.libcore.javabean;

import com.ashlikun.utils.other.StringUtils;

/**
 * @author　　: 李坤
 * 创建时间: 2018/7/18 13:15
 * 邮箱　　：496546144@qq.com
 * <p>
 * 功能介绍：键值对的类
 */

public class KeyAndValus {

    private int key;
    private String keyS;
    private String valus;

    public KeyAndValus(int key, String valus) {
        this.key = key;
        this.valus = valus;
    }

    public KeyAndValus() {
    }

    public int getKey() {
        return key;
    }

    public String getValus() {
        return StringUtils.dataFilter(valus);
    }

    public String getKeyS() {
        return StringUtils.dataFilter(keyS);
    }

}

File: component_core/src/main/java/com/ashlikun/baseproject/libcore/javabean/KeyAndValus.kt
package com.ashlikun.baseproject.libcore.javabean

import com.ashlikun.utils.other.StringUtils

/**
 * @author　　: 李坤
 * 创建时间: 2018/7/18 13:15
 * 邮箱　　：496546144@qq.com
 *
 *
 * 功能介绍：键值对的类
 */

class KeyAndValus(key: Int = 0, valus: String? = null) {
    private var key = 0;
    private var keyS: String? = null
        get() = StringUtils.dataFilter(keyS)
    private var valus: String? = null
        get() = StringUtils.dataFilter(valus)

    init {
        this.key = key;
        this.valus = valus;
    }
}
